src/api/app/controllers/webui/request_controller.rb

```diff
@@ -324,6 +325,16 @@ def mentioned_issues
     @active = 'mentioned_issues'
   end
 
+  def render_diff
```

Comment:
It should be removed because the action has been moved to the comments controller.